src/random.h

```diff
@@ -12,6 +12,14 @@
 
 #include <stdint.h>
 
+#ifdef _MSC_VER
+#ifdef _WIN64
+typedef int64_t ssize_t;
+#else
+typedef int32_t ssize_t;
+#endif
+#endif
+
```

Comment:
I think just making MAX_TRIES an int should remove the need for this?

Reply:
> I think just making MAX_TRIES an int should remove the need for this?

Yes this would be better than adding a top-level definition that could conflict with other libraries. (Also, if we did need a top-level definition, compat.h would probably be a better place to put it.)

Reply:
@ryanofsky I agree it's very clunky to add yet another ssize_t typedef. I did consider adding to compat.h, see the main PR comment. Happy to go with whatever the consensus is.

Reply:
> Happy to go with whatever the consensus is.

I think you should just make MAX_TRIES an int like Cory suggested.